### ISSUE TITLE
TensorFlowEvent metrics collector doesn't find events written by torch / tensorboardX

### ISSUE THREAD

**Issue by garymm** [kubeflow/katib#2466]
### What happened?

Trial code writes tensorboard summary using tensorboardX or torch.utils.tensorboard.
Experiment configured to use TensorFlowEvent metrics collector.
The metrics collector reports the metric is unavailable.

### What did you expect to happen?

I expect the metrics collector to report the metric accurately.

### Environment

Katib v0.17.0

### Impacted by this bug?

Give it a 👍 We prioritize the issues with most 👍
